cmd/dep/prune.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2016 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/golang/dep"
+	"github.com/sdboyer/gps"
+	"github.com/sdboyer/gps/pkgtree"
+
+	"github.com/pkg/errors"
+)
+
+const pruneShortHelp = `Prune the vendor tree of unused packages`
+const pruneLongHelp = `
+Prune is used to remove unused packages from your vendor tree.
```

Comment:
OK, I now just have one docs nit...let's please have this say the following:

```
Prune is used to remove unused packages from your vendor tree.

STABILITY NOTICE: this command creates problems for vendor/ verification. As such, it may be removed and/or moved out into a separate project later on.
```

I just want to warn people everywhere we can 😄